### ISSUE TITLE
Allow NopSelectTagHelper to work with string values

### ISSUE THREAD

**Issue by alinmircea** [nopSolutions/nopCommerce#4725]
_The model item passed into the ViewDataDictionary is of type 'System.String[]', but this ViewDataDictionary instance requires a model item of type 'System.Collections.Generic.List`1[System.Int32]'.'_

At the moment a model property of type `List<string>` cannot be bound to MultiSelect.cshtml because the view is expecting a model of type `List<int>`.

Removing `"@model List<int>"` form line 1 of MultiSelect.cshtml will allow using `<nop-select/>` with a string collection.

I expect no other change is required because in the context of using `List<string>` as the datasource you would not expect a value of "0", so the javascript code in the MultiSelect.cshtml can remain as it is and will apply only to those situations where values are numeric. This way it retains backwards compatibility with the current functionality.

The only unforeseen case would be when using a `List<string>` where one of the values is 0, but that does not impact anything that currently exists. There is no functionality in the framework or in any existing plugin that would be impacted because `List<string>` collections are not currently allowed. 

For consistency, consider changing the javascript to such that it also accounts for empty string values, not only for "0"

The second potential impact of making this change is that it would allow any property into the view, not only `List<int>, List<string>`. One way to handle that might be to simply check it in Nop.Web.Framework.TagHelpers.Admin.NopSelectTagHelper along with if(multiple)

```
if (multiple)
{
        selectList = _htmlHelper.Editor(tagName, "MultiSelect", new {htmlAttributes, SelectList = Items});
}
```

That is starting to look ugly so, the cleaner solution to this would be to duplicate MultiSelect.cshtml into MultiSelectString.cshtml and use it when the property type is `collection<string>` while allowing everything else to flow into MultiSelect.cshtml  just like it currently does.

**Comment by alinmircea** [comment]
This situation was encountered when integrating with an external system, where saving all external data into the database locally doesn't make sense and there is still a need to preset the user with specific forms where they are able to pick specific options (which are string in this case)

**Comment by DmitriyKulagin** [comment]
Closed #4725
